client/src/App.scss

```diff
@@ -0,0 +1,89 @@
+%screen-centering{
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.App{
+  height: 100vh;
+  display: flex;
+  color: white;
+}
+
+.App_aside{
+  width: 50%;
+  @extend %screen-centering;
+  // display: flex;
+  // justify-content: center;
+  // align-items: center; 
+}
+
+.App_form{
+  width:  50%;
+  overflow: auto;
+  background-image: url('./images/07cc6abd390ab904abbf31db5e6ea20357f8b127.png');
+  background-size: cover;
+  background-position: center;
+}
+
+
+
+
+h1{
+  font-size: 350%;
+  color: black;
+}
+
+
+
+
+.info-box{
+  width: 70%;
+  height: 70%;
+  border: 1px solid grey;
+}
+
+.center{
+  text-align: center;
+}
+
+.mb-0{
```

Comment:
These are strange classes names... and can we leverage material-ui way of doing styling:
https://material-ui.com/styles/basics/

I generally like to use HOC method or Hooks (if you are using functional components). 
In addition, a lot of the material-ui components have fields like `paragraph` that can be used for spacing